doc/dev/crimson/crimson.rst

```diff
@@ -176,6 +176,68 @@ see `Prometheus`_ for more details.
 Profiling Crimson
 =================
 
+fio
+---
+
+``crimson-store-nbd`` exposes crimson object store internals as an NBD server
```

Comment:
Is the `internals` word is necessary? Maybe *crimson object store* or even *crimson's `FuturizedStore` as an NBD server* would be fine?

Reply:
@rzarzynski i changed it to "SeaStore" in hope that it is more specific. IMHO, `FuturizedStore` is an interface which can imply two implementations, but crimson-store-nbd only exercises SeaStore.

Reply:
@rzarzynski sorry, i was wrong. updated and repushed.